api/envoy/config/filter/http/router/v2/router.proto

```diff
@@ -63,4 +63,9 @@ message Router {
       "x-envoy-retry-on"
     ]
   }];
+
+  // If not set, ingress Envoy will ignore
```

Comment:
nit: prefer positive vs. double negative. 

"If true, the ingress Envoy will use ..."

Reply:
@mattklein123 in previous review cycle there was a suggestion to describe the behaviour when flag is not set, as this way we don't need to update comment when timeout decision logic changes.

Reply:
OK if this was already discussed no problem.